documentation/install_from_packages.markdown

```diff
@@ -7,103 +7,58 @@ canonical: "/puppetserver/latest/install_from_packages.html"
 
 > If you're also using PuppetDB, check its [requirements](https://puppet.com/docs/puppetdb/latest/index.html#system-requirements).
 
-## Platforms with Packages
-
-Puppet provides official packages that install Puppet Server 5.1 and all of its prerequisites on x86_64 architectures for the following platforms, as part of [Puppet Platform][repodocs].
-
-### Red Hat Enterprise Linux
-
--   Enterprise Linux 6
--   Enterprise Linux 7
-
-### Debian
-
--   Debian 8 (Jessie)
-
-Java 8 runtime packages do not exist in the standard repositories for Debian 8 (Jessie).  To install Puppet Server on Jessie, [configure the `jessie-backports` repository](https://backports.debian.org/Instructions/), which includes openjdk-8:
-
-```
-echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
-apt-get update
-apt-get -t jessie-backports install "openjdk-8-jdk-headless"
-apt-get install puppetserver
-```
-
-If you upgraded on Debian from older versions of Puppet Server, or from Java 7 to Java 8, you must also configure your server to use Java 8 by default for Puppet Server 5.x:
-
-```
-update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
-```
-
-### Ubuntu
-
--   Ubuntu 18.04 (Bionic)
--   Ubuntu 16.04 (Xenial)
-
-On Ubuntu 18.04, enable the [universe repository](https://help.ubuntu.com/community/Repositories/Ubuntu), which contains packages necessary for Puppet Server.
-
-### SUSE Linux Enterprise Server
-
--   SLES 12 SP1
-
-## Quick Start
+## Install Puppet Server from packages
 
 1.  [Enable the Puppet package repositories][repodocs], if you haven't already done so.
-2.  Stop the existing Puppet master service. The method for doing this varies depending on how your system is set up.
-
-    If you're running a WEBrick Puppet master, use: `service puppetmaster stop`.
-
-    If you're running Puppet under Apache, you'll instead need to disable the puppetmaster vhost and restart the Apache service. The exact method for this depends on what your Puppet master vhost file is called and how you enabled it. For full documentation, see the [Passenger guide][passengerguide].
-
-    -   On a Debian system, the command might be something like `sudo a2dissite puppetmaster`.
-    -   On RHEL or CentOS systems, the command might be something like `sudo mv /etc/httpd/conf.d/puppetmaster.conf ~/`. Alternatively, you can delete the file instead of moving it.
-
-    After you've disabled the vhost, restart Apache, which is a service called either `httpd` or `apache2`, depending on your OS.
 
-    Alternatively, if you don't need to keep the Apache service running, you can stop Apache with `service httpd stop` or `service apache2 stop`.
-
-3.  Install the Puppet Server package by running:
+2.  Install the Puppet Server package by running:
 
         yum install puppetserver
 
-    Or
+    or
```

Comment:
@justinstoller When would you use one or the other? Would the user know which to use?

Reply:
Yeah, the preferred package manager depends on the "family" the OS belongs to. This list `yum` & `apt` is actually incomplete (SLES uses `zypper`, iirc) however. I expect users that are doing this level of system administration to know how to install packages and grok which one (if either) is applicable.